### PR TITLE
Fix quasar vite setup using v2

### PR DESCRIPTION
**Pull request by efonte**

### CODE REVIEW

File: src/install.js [hunk 1]
```diff
@@ -14,4 +24,7 @@ module.exports = function (api) {
   api.extendJsonFile('.vscode/extensions.json', {
     recommendations: ['apollographql.vscode-apollo'],
   })
+
+  // We extend /quasar.conf.js
+  api.extendQuasarConf(extendConf)
```
Comment (yusufkandemir) [inline src/install.js:29]: This is not needed and will not do anything. Install only runs once, but we need the boot file on every run, which is already handled in the `index.js` script. It also handles SSR and such. So, please remove it.

**Review by yusufkandemir — CHANGES_REQUESTED** [review]
Thanks for the contribution. I am requesting some changes.

**Comment by yusufkandemir** [comment]
This PR fixes #119.

**Comment by ricar2ce** [comment]
When will this change be applied?

File: src/install.js (hunk 1 above)
Comment (smolinari) [reply src/install.js:29]: @efonte - Can you reply/ take action please?

Scott

**Comment by smolinari** [comment]
> When will this change be applied?

I'll have this merged and released this coming weekend at the latest. 

Scott

**Comment by rowild** [comment]
Until this is merged, how can I enable this extension anyway? Manually overwriting won't help, because yarn will always install the npm version. And there isno other way to initiate the installation procedure... Any tips?

**Comment by rowild** [comment]
@smolinari Thank you for mergin! But unfortunately I still get an error:

 Extension(@quasar/apollo): Dependency not found - @quasar/app. Please install it.
 
 To my knowledge it should either be `app-webpack` or `app-vite`, but `app` does not exist anymore...

**Comment by smolinari** [comment]
@rowild - Please try again. The tagging got borked between the two versions.

Scott

**Comment by rowild** [comment]
@smolinari Thank you for your quick reply!
Unfortunately the error remains., even though now beta.4 gets installed. I tried in a new app setup with quasar-cli choosing "vite" and in my existing project.

Bildschirmfoto 2022-06-26 um 10 48 47

This is what the package tree in node_modules/@quasar looks like:

Bildschirmfoto 2022-06-26 um 10 53 16

I would like to test locally, but I fail to di a "npm link" on "quasar-app-extension-apollo", which I moved to the Desktop (and uninstalled in the project).

**Comment by smolinari** [comment]
@rowild - It should be working now with beta5. Thanks for the quick feedback. 

Scott

**Comment by rowild** [comment]
@smolinari Confirmed - it is working now! Juhuu!
May I ask what needed to be changed? I can't see any differences to the previous PR...

**Comment by smolinari** [comment]
@rowild - Somehow I had published unchanged code. I had to republish only, but to do that, I had to bump the version. 

Scott

**Pull request merged**